oa-enterprise/lib/omniauth/strategies/saml/auth_request.rb

```diff
@@ -9,7 +9,7 @@ class SAML
       class AuthRequest
         
         def create(settings, params = {})
-          uuid = "_" + UUID.new.generate
```

Comment:
Is this maybe here to abide by some saml spec?

Reply:
Yep: see the definition of [`NCName`](https://www.w3.org/TR/1999/REC-xml-names-19990114/#NT-NCName) — it must start with a letter or underscore, so we just throw an underscore before the UUID in case it begins with a digit.